docs/source/markdown/podman-auto-update.1.md.in

```diff
@@ -7,50 +7,47 @@ podman\-auto-update - Auto update containers according to their auto-update poli
 **podman auto-update** [*options*]
 
 ## DESCRIPTION
-**podman auto-update** looks up containers with a specified `io.containers.autoupdate` label (i.e., the auto-update policy).
+**podman auto-update** pulls down new container images and restarts containers configured for auto updates.
+To make use of auto updates, the container or Kubernetes workloads must run inside a systemd unit.
+After a successful update of an image, the containers using the image get updated by restarting the systemd units they run in.
+Please refer to `[Quadlet](podman-systemd.unit.5.md)` on how to run Podman under systemd.
```

Comment:
The `[Quadlet]` thing bothers me. First: in generated HTML, it does not do what you want. Because of the backticks, it shows up as actual bracket-Quadlet-bracket-paren-etc-etc. Solution: remove the backticks.

Even then, though, that does not yield anything useful in the \*roff man page:
>Please refer to [Quadlet] on how ...

...but I then search in-page for "quadl" and get no actual hits. How do I learn about "quadlets"? 

So, suggestion:
>Please refer to `quadlet(5)` for more on how to run Podman under systemd.

That means we don't get a nice link in the HTML docs, but it seems like the least-awful tradeoff overall.

Reply:
Also, ISTR that backticks would produce differently-colored output in man pages? I see no visual distinction any more. Am I misremembering?

Reply:
Nice suggestion, thanks!